sdk/translation/azure-ai-translation-document/azure/ai/translation/document/_helpers.py

```diff
@@ -44,14 +45,39 @@ def get_translation_input(args, kwargs, continuation_token):
             target_language_code = kwargs.pop("target_language_code", None)
             if not target_language_code:
                 target_language_code = args[2]
+
+            # Additional kwargs
+            source_language_code = kwargs.pop("source_language_code", None)
+            prefix = kwargs.pop("prefix", None)
+            suffix = kwargs.pop("suffix", None)
+            storage_type = kwargs.pop("storage_type", None)
+            source_storage_source = kwargs.pop("source_storage_source", None)
```

Comment:
just to make sure: are you getting the "AzureBlob" default from the generated code / service, and that's why you can default to None here?

Reply:
yes service defaults to AzureBlob